sematic/ui/src/pipelines/PipelinePanelsContext.tsx

```diff
@@ -6,6 +6,10 @@ export const PipelinePanelsContext = React.createContext<{
     setSelectedPanelItem: (panelItem: string) => void;
     selectedRun: Run | undefined;
     setSelectedRunId: (runId: string) => void;
+    selectedRunTab: string;
+    setSelectedRunTab: (runTab: string) => void;
+    selectedArtifactId: string;
+    setSelectedArtifactId: (artifactName: string) => void;
```

Comment:
Due to the series of discussions, I guess we should name it to `setSelectedArtifactName` now :)